### ISSUE TITLE
Improve cosmetic appearance of HTML

### ISSUE THREAD

**Issue by atreya2011** [adonovan/spaghetti#2]
Would improving the cosmetic appearance of HTML be just adding CSS to the existing HTML file? If my understanding is correct, introducing a complicated build process for generating static assets is not required right? I was considering perhaps making a PR where I just link an external minified CSS of a CSS framework/library (perhaps from a CDN) and using classes present with the CSS framework/library to improve the cosmetic appearance of the HTML.

**Comment by adonovan** [comment]
I'm sure you can appreciate the irony of a dependency elimination tool itself having unnecessary dependencies, so I would prefer not to import other packages solely for cosmetic reasons. I imagine that someone with much more DOM/CSS knowledge than me could easily spruce up the text, layout, and perhaps some of the JS interactions without deeper changes.

**Comment by atreya2011** [comment]
I totally understand. So just extra CSS to improve the appearance without any additional CSS library/framework dependencies. I did notice that you are using jQuery to spruce up the JS interactions? Would you mind them being replaced by vanilla JS?

**Comment by adonovan** [comment]
Good point, the client-side code does already have a dynamic dependency on jQuery from a CDN. I suppose adding a similar dependency for CSS would be fine, if the CSS project has a good track record and doesn't have additional dependencies of its own. Do you have a particular one in mind?

**Comment by atreya2011** [comment]
I had the following two in mind. Both of them have a good track record and a minified CSS available via a CDN.

1. [Bootstrap](https://getbootstrap.com/docs/4.6/getting-started/introduction/)
2. [Bulma](https://bulma.io/documentation/) (CSS classes are a bit more semantic and easier to read personally)

I have used both of them in personal and professional projects.

**Comment by adonovan** [comment]
Thanks; these both fit the bill. Take your pick.

**Comment by atreya2011** [comment]
Thanks. Will most probably go ahead with Bootstrap since that’s been around the longest time.
